test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -17,16 +17,27 @@ def test_stray_files(image):
     container_name = f"test_tmp_dirs-{image.split('/')[-1].replace('.', '-').replace(':', '-')}"
     _start_container(container_name, image, ctx)
 
-    # Running list of artifacts we do not want in any of the directories
-    stray_artifacts = ["deep-learning-containers.py"]
+    # Running list of artifacts/artifact regular expressions we do not want in any of the directories
+    stray_artifacts = [r"\.py"]
 
-    # Run assertions for each directory
+    # Run assertions for tmp file directory
     tmp = _run_cmd_on_container(container_name, ctx, "ls -A /tmp")
     _assert_artifact_free(tmp, stray_artifacts)
+    allowed_tmp_files = ("hsperfdata_root", ".", "..")
```

Comment:
Nit: for more clarity, move stray_artifacts and allowed_tmp_files closer for colocation. Why is allowed_tmp_files a tuple while stray_artifacts a list?